### ISSUE TITLE
[feat] docker compose up --build -d

### ISSUE THREAD

**Issue by looles** [YuqiHuai/SORA-SVL#83]
![image](https://github.com/user-attachments/assets/ca50fe76-986a-42b8-b22b-c509bf87b593)
Hello Dr Huai, thank you very much for doing this work, I am having a problem with the installation as shown in the picture above, can you help me out? Thank you very much for your help!

**Comment by YuqiHuai** [comment]
Hi, thanks for posting this issue. Looks like commands to install appropriate dependency was unsuccessful. Are you running this from China?
——
I am also traveling in China right now therefore unable to spend too much effort on resolving this. But I expect this to be resolved if a VPN is used.

**Comment by looles** [comment]
Yes, currently experiencing a lot of problems because of this, thanks to Dr Huai for his help!

**Comment by vit271828** [comment]
I also encountered the same problem, please tell me how to solve it.

**Comment by YuqiHuai** [comment]
Looks like this has became a common issue and many of you have managed to solve it.
